### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'start[0].x')

### ISSUE THREAD

**Issue by TheNomadInOrbit** [codesinghanoop/react-native-d3-tree-graph#10]
Hi! I'm working on this example and getting this error below which is coming from **siblingUtil.js**
**TypeError: undefined is not an object (evaluating 'start[0].x')**
Please help me out.

![Screenshot_20200520-082013_Expo](https://user-images.githubusercontent.com/36538038/82401355-08eeb680-9a73-11ea-9e27-678a10359be1.jpg)

**Comment by codesinghanoop** [comment]
@itsusmanmirza are you running example in this repo ?

**Comment by TheNomadInOrbit** [comment]
No not in this repo I'm importing this in my own project, in Home Screen.tsx, and yes I'm using TypeScript as well.

**Comment by codesinghanoop** [comment]
are you already having react-native-svg version in your project before integrating this library ? @itsusmanmirza

**Comment by TheNomadInOrbit** [comment]
Yes react-native-svg was already installed and I'm using this version.
![Code_8fM7XZ5we1](https://user-images.githubusercontent.com/36538038/83002942-12ee5780-a027-11ea-8daf-b88b9f5a801e.png)

**Comment by codesinghanoop** [comment]
@itsusmanmirza  got it.. In you project you have to use the same version as this library is using..Please try and let me know if you face any issue

**Comment by TheNomadInOrbit** [comment]
Alright I'll update you soon.

**Comment by TheNomadInOrbit** [comment]
have tried, no luck, not working.

**Comment by codesinghanoop** [comment]
@itsusmanmirza can you please share the steps you followed .

**Comment by codesinghanoop** [comment]
@itsusmanmirza please share your package.json

**Comment by codesinghanoop** [comment]
Due to inactivity closing this for now.

**Comment by kr396** [comment]
There is problem in below code (which is given in readme). If we run below code it is shoving TypeError: undefined is not an object (evaluating 'start[0].x'). kindly run example it is working fine,

@codesinghanoop , kindly remove or update below code in readme file. Thank You @codesinghanoop  nice work.

import AwesomeHierarchyGraph from 'react-native-d3-tree-graph'

var root = {
    name: "",
    id: 1,
    hidden: true,
    children: [ {
            name: "Q",
            id: 16,
            no_parent: true
        }]
    }

var siblings = [{
    source: {
        id: 3,
        name: "C"
    },
    target: {
        id: 11,
        name: "K"
    }
 }]
 
export default class example extends Component {
  render() {
    return (
        <View style={styles.container}>
        <AwesomeHierarchyGraph
         root = {root}
         siblings = {siblings}
        />
       </View>
    );
  }

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
  instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
  },
});

**Comment by michaeljamieson10** [comment]
I have the same issue

**Comment by bishalvaii** [comment]
@codesinghanoop facing this issue . Do you have any idea how to solve this?

**Comment by codesinghanoop** [comment]
@michaeljamieson10  & @bishalvaii , Run example , Please pick code from readme.md, i will update it later. Let me know if it doesn't works for you
